docs/.templates/cli.md

```diff
@@ -106,7 +106,15 @@ mv tauri.sh /usr/local/etc/bash_completion.d/tauri.bash
 ```
 
   </TabItem>
-    <TabItem value="Cargo">
+  <TabItem value="Bun">
+
+```shell
+bunx tauri completions --shell bash > tauri.sh
```

Comment:
`bunx` will use a package directly from npm and in this case it will try to use the alpha version of tauri which is deprecated so we need to use `bun run` to use the locally installed CLI
```suggestion
bun run tauri completions --shell bash > tauri.sh
```

Reply:
Like `npx`, `bunx` will use the locally installed version if it exists